src/main/webapp/scripts/ou-script.js

```diff
@@ -1,4 +1,7 @@
-function getAllOUs(token){
```

Comment:
This is the OAuth token? Is oauth2-test-params a variable name you all created? Possibly remove test in a future cl if so.

Reply:
Yes, so in the new commit, I added a variable token fetching from params['access_token'] at the beginning of the script